src/www/build.action.sh

```diff
@@ -32,10 +32,13 @@ done
 WEBPACK_CONFIG=
 case "${PLATFORM}" in
   windows|linux)
-    WEBPACK_CONFIG="src/www/webpack_electron.js"
+    WEBPACK_CONFIG="src/www/webpack_electron.mjs"
```

Comment:
Why do you need to change the extension?

Reply:
this tells node the file is an es module so we can use import/export syntax. there are other ways we can do this https://nodejs.org/api/esm.html - but just using `mjs` is the most incremental and the least disruptive to our scripts